components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -2406,6 +2406,9 @@ private ConfigParameters() {
         public static final String DEFAULT_KEY_MANAGER = "Resident Key Manager";
         public static final String DEFAULT_KEY_MANAGER_TYPE = "default";
         public static final String DEFAULT_KEY_MANAGER_DESCRIPTION = "This is Resident Key Manager";
+        public static final String IS_EXTERNAL_IDP = "is_external_idp";
```

Comment:
where do we use this ?

Reply:
This has been left due to a previous change that I have done. Thanks for pointing this out. Removed via 378da6c1483201083cd5bd67180950634920f091